src/components/EventsForm.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+  Alert,
+  Button,
+  DatePicker,
+  Form,
+  Input,
+  InputNumber,
+  Upload,
+} from 'antd';
+import { Moment } from 'moment';
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import styled from 'styled-components';
+import { UserAuthenticationReducerState } from '../auth/ducks/types';
+import {
+  asyncRequestIsFailed,
+  asyncRequestIsLoading,
+} from '../utils/asyncRequest';
+import { FileField } from '../utils/fileEncoding';
+import FormContainer from './FormContainer';
+
+const { Dragger } = Upload;
+const { TextArea } = Input;
+
+const PaddedAlert = styled(Alert)`
+  margin-top: 1em;
+  margin-bottom: 2em;
+`;
+
+export interface EventsFormData {
```

Comment:
Would it be possible to extend from another event type that has title, capacity, thumbnail, etc?